cEP-0004.md

```diff
@@ -35,6 +35,24 @@ The following changes shall be made to the UI:
 
 The below sections describe why and how we should make these changes.
 
+### Mockup
+
+```
+test.py
+|   1| print(•1•)
+|    | [NORMAL] PEP8Bear:
+|    | The code does not comply to PEP8.
+|----|    | /tmp/test.py
+|    |++++| /tmp/test.py
+|   1|    |-print( 1 )
+|    |   1|+print(1)
+|    |
+|    | * Do (n)othing
+|    |   (O)pen file(s)
+|    |   (A)pply the patch
+|    | Choose an action:
+```
+
 ### Remove most of the text from the action selection screen
```

Comment:
i.e. if it's not a renaming merge the two lines both containing the file name into one?